network/msgOfInterest.go

```diff
@@ -87,3 +87,13 @@ func MarshallMessageOfInterestMap(tagmap map[protocol.Tag]bool) []byte {
 	topics := Topics{Topic{key: "tags", data: []byte(tags)}}
 	return topics.MarshallTopics()
 }
+
+// MessageOfInterestMaxSize returns the maximum size of a MI message sent over the network
+// by encoding all of the tags currenttly in use.
+func MessageOfInterestMaxSize() int {
```

Comment:
This should be in the test file, since it is only used from the test.

Reply:
I can move it but would have to move it to `node_test.go` which seems like an odd place for it and it would break symmetry since all other MaxSize() functions are available in non-test code.